python/test/t_CovarianceModel_std.py

```diff
@@ -61,6 +62,20 @@ def test_model(myModel, test_grad=True, x1=None, x2=None):
         print('dCov/dP=', pGrad)
         ot.PlatformInfo.SetNumericalPrecision(precision)
 
+def fitKriging(covarianceModel):
```

Comment:
should be moved to origins test or create a new test

Reply:
What is the "origins" test?

Reply:
the test is designed for value predicted, gradient.. `CovarianceModel` is in `Base` so even for the tests, spirit is to rely only on classes from `Base`
We might have more elaborated `kriging` tests (for example, a `t_KrigingAlgorithm_isotropic_std`)